src/Features/CSharp/Portable/ImplementInterface/CSharpImplementInterfaceService.cs

```diff
@@ -40,10 +40,9 @@ public CSharpImplementInterfaceService()
                         var interfaceSymbolInfo = model.GetSymbolInfo(interfaceNode, cancellationToken);
                         if (interfaceSymbolInfo.CandidateReason != CandidateReason.WrongArity)
                         {
-                            var interfaceType = interfaceSymbolInfo.GetAnySymbol() as INamedTypeSymbol;
                             cancellationToken.ThrowIfCancellationRequested();
 
-                            if (interfaceType != null && interfaceType.TypeKind == TypeKind.Interface)
+                            if (interfaceSymbolInfo.GetAnySymbol() is INamedTypeSymbol interfaceType && interfaceType.TypeKind == TypeKind.Interface)
```

Comment:
any risk around changing the place where we ThrowIfCancellationRequested?

Reply:
should be ok :)